Whiskey/en-US/about_Whiskey_SetVariableFromPowerShellDataFile.help.txt

```diff
@@ -0,0 +1,90 @@
+TOPIC
+    about_Whiskey_about_Whiskey_SetVariableFromPowerShellDataFile_Task
+
+DESCRIPTION
+    The `SetVariableFromPowerShellDataFile` task creates variables from data in a PowerShell data file (e.g. a .psd1 file, a module manifest, etc.). It uses the `Import-PowerShellDataFile` cmdlet, which was introduced in PowerShell 5. Set the `Path` property to the path to the PowerShell data file/manifest to read. 
+    
+    Set the `Variables` property to a key/value mapping of properties from the manifest file you want as variables. The key of each mapping should be the name of the property from the data file. The value of each mapping should be a variable name. A variable with that name will be created with that property's value. 

```

Comment:
This is by design. It's unfortunate, but since there's no query language for data files, the simplest solution was to mimic the structure between the two files.